is.js

```diff
@@ -384,7 +384,7 @@
         if (is.not.string(str)) {
             return false;
         }
-        var words = str.split(' ');
+        var words = str.split(' ').filter(function(el) {return el.length != 0});
```

Comment:
Let's avoid using `Array#filter` here and instead move the check into the `for` loop below.